core/spec.md

```diff
@@ -427,9 +446,27 @@ be one of the following data types:
   [RFC 1738](https://datatracker.ietf.org/doc/html/rfc1738).
 
 All relative URIs and URLs that reference entities within the Registry MUST
-begin with `/` and MUST be relative to the root path of the Registry service.
+begin with `/`. With the exception of `xid` values, relative URIs/URLs that
+reference entities that are part of a document view `GET` response (`?doc`
+is enabled), MUST be relative to the path specified on the request. Relative
```

Comment:
I don't understand this... if it's meant to be "relative to the path" why would it start with a "/"?

My understanding is:

- If the URL begins with a "/", it's an absolute *path*, but relative to the originating server
- If a URL *doesn't* begin with a "/" then it's a relative path

So a "relative URL beginning with a `/` but relative to the path specified on the request" sounds like a contradiction in terms to me.